lib/matplotlib/axes/_axes.py

```diff
@@ -4916,6 +4929,8 @@ def reduce_C_function(C: array) -> float
             DATA_PARAMETER_PLACEHOLDER
 
         **kwargs : `~matplotlib.collections.PolyCollection` properties
+            If *xscale* or *yscale* is set to 'symlog',
+            additional `~matplotlib.scale.SymmetricalLogTransform` parameters.
```

Comment:
```suggestion
            If *xscale* or *yscale* is set to 'symlog',
            additional `~matplotlib.scale.SymmetricalLogTransform` parameters.
```

This is not correct anymore. kwargs have moved to the scale parameters